packages/next-tinacms-cloudinary/src/cloudinary-media-store.ts

```diff
@@ -44,14 +44,21 @@ export class CloudinaryMediaStore implements MediaStore {
       throw new Error(responseData.message)
     }
     const fileRes = await res.json()
-
-    // TODO: be programmer
-    // NOTE: why do we need this?
     await new Promise((resolve) => {
       setTimeout(resolve, 2000)
     })
+    /**
+     * Format the response from Cloudinary to match Media interface
+     */
+    const parsedRes: Media = {
+      type: fileRes.resource_type === 'image' ? 'file' : 'dir',
```

Comment:
is this correct? How do non-image assets (video, audio, text) come through the API? I wouldn't want to handle those as directories

Reply:
Good catch, I'd need to check Cloudinary docs on how they handle that, but agree that check isn't robust enough